homeassistant/components/google_weather/strings.json

```diff
@@ -61,5 +64,60 @@
         }
       }
     }
+  },
+  "entity": {
+    "sensor": {
+      "apparent_temperature": {
+        "name": "[%key:component::weather::entity_component::_::state_attributes::apparent_temperature::name%]"
+      },
+      "cloud_coverage": {
+        "name": "[%key:component::weather::entity_component::_::state_attributes::cloud_coverage::name%]"
+      },
+      "dew_point": {
+        "name": "[%key:component::weather::entity_component::_::state_attributes::dew_point::name%]"
+      },
```

Comment:
Generally we don't want platform A to use strings of a part of platform B. I now see that open weather map does the same, but it should not IMO